### DISCUSSION TITLE
For Apple Silicon users possibly document enabling Docker beta feature - Rosetta for Linux - results in noticeably faster local runs on the dummy data

### DISCUSSION THREAD

**Discussion by remlapmot** [opensafely/documentation#1097] [General]
This is just for users with Apple Silicon Macs.

Docker 4.16.0 brought a new beta feature - Rosetta for Linux.

It's off by default. Once enabled in Docker settings

<img width="1228" alt="screen-10" src="https://user-images.githubusercontent.com/3777473/212496386-39acf1d3-ade7-4e13-bf52-ccdaf2a35e0d.png">

it gives a nice speed increase running the dummy data locally. For example, in our [cox-ipw](https://github.com/opensafely-actions/cox-ipw/) action running the tests with
```
opensafely run run_all -f
```
take under 1 minute with this feature enabled; with this feature disabled they take about 3 minutes.

So this would possibly be worth mentioning in the documentation site.

**Comment by tomodwyer** [comment]
Thanks for flagging this @remlapmot - we've got a couple of M1 users in the team, so we'll take a look as this feature.

Once it's out of beta, we'll add it to the documentation.

**Reply by remlapmot** [reply]
Just noting in last week's release of Docker version 4.25.0, that it's out of beta and it seems to be enabled by default (which is good).

<https://docs.docker.com/desktop/release-notes/#4250>

**Reply by tomodwyer** [reply]
Brilliant, thanks for letting me know. I've added an issue and we'll get the docs updated.

**Comment by remlapmot** [comment]
Thanks for doing this.

Just pointing out that #1098 was a duplicate issue, so you can close that one as well.

**Reply by StevenMaude** [reply]
Done; thanks!
